parachains/common/src/xcm_config.rs

```diff
@@ -64,7 +64,7 @@ impl ShouldExecute for DenyReserveTransferToRelayChain {
 			message.0.iter().any(|inst| matches!(inst, ReserveAssetDeposited { .. }))
 		{
 			log::warn!(
-				target: "xcm::barrier",
+				target: "xcm::barriers",
```

Comment:
`barrier` is fine, we don't need to change this.

Reply:
I thought that this is a typo, because everywhere (polkadot, cumulus repos) is used just `target: "xcm::barriers"`,
e.g. other `impl ShouldExecute for DenyReserveTransferToRelayChain` in Cumulus have: `target: "xcm::barriers"`:

https://github.com/paritytech/cumulus/blob/master/parachain-template/runtime/src/xcm_config.rs#L146
https://github.com/paritytech/cumulus/blob/master/parachains/runtimes/testing/penpal/src/xcm_config.rs#L206

@KiChjang 
so, I guess, we should better keep it "in the same way" at least for Cumulus,
so, there are several possibilities, wdyt?
1. just revert this change, and keep different `xcm::barrier` and `xcm::barriers` in cumulus
2. just for Cumulus, use everywhere ``xcm::barrier`
3. also for Cumulus, use everywhere ``xcm::barriers`

Reply:
Okay, if cumulus has been using `barriers`, then let's keep it that way.